README.md

```diff
@@ -16,16 +16,15 @@ To quickly get the Deep Log Inspection server running, `cd` into `log-server` an
 Assuming all goes well, the following services should be exposed on your host
 machine:
 
- * monasca-log-api on port 8090
- * elasticsearch on ports 9200 and 9300
- * kibana on port 443
+ * monasca-log-api on port 443
+ * elasticsearch on port 9200
+ * kibana on port 80
```

Comment:
Isn't it weird to have 2 diff services in the ports typically used for http and https?
http -> kibana
https -> monasca-log-api

Reply:
ignore this, it's and old version that will has been updated in the docs branch